flexget/plugins/metainfo/imdb_lookup.py

```diff
@@ -448,6 +449,7 @@ def _parse_new_movie(self, imdb_url, session):
         movie.score = parser.score
         movie.votes = parser.votes
         movie.year = parser.year
+        movie.runtime = parser.runtime
```

Comment:
Is this going to be added to the db table, and did you upgraded the database schema